base/compiler/tfuncs.jl

```diff
@@ -384,29 +408,30 @@ add_tfunc(typeassert, 2, 2,
               end
               return typeintersect(v, t)
           end, 4)
-add_tfunc(isa, 2, 2,
-          function (@nospecialize(v), @nospecialize(t))
-              t, isexact = instanceof_tfunc(t)
-              if !has_free_typevars(t)
-                  if t === Bottom
-                      return Const(false)
-                  elseif v ⊑ t
-                      if isexact
-                          return Const(true)
-                      end
-                  elseif isa(v, Const) || isa(v, Conditional) || isdispatchelem(v)
-                      # this tests for knowledge of a leaftype appearing on the LHS
-                      # (ensuring the isa is precise)
-                      return Const(false)
-                  elseif isexact && typeintersect(v, t) === Bottom
-                      if !iskindtype(v) #= subtyping currently intentionally answers this query incorrectly for kinds =#
-                          return Const(false)
-                      end
-                  end
-              end
-              # TODO: handle non-leaftype(t) by testing against lower and upper bounds
-              return Bool
-          end, 0)
+
+function isa_tfunc(@nospecialize(v), @nospecialize(t))
+    t, isexact = instanceof_tfunc(t)
+    if !has_free_typevars(t)
+        if t === Bottom
+            return Const(false)
+        elseif v ⊑ t
+            if isexact
+                return Const(true)
+            end
+        elseif isa(v, Const) || isa(v, Conditional) || isdispatchelem(v)
+            # this tests for knowledge of a leaftype appearing on the LHS
+            # (ensuring the isa is precise)
+            return Const(false)
+        elseif isexact && typeintersect(v, t) === Bottom
+            if !iskindtype(v) #= subtyping currently intentionally answers this query incorrectly for kinds =#
+                return Const(false)
+            end
+        end
+    end
+    # TODO: handle non-leaftype(t) by testing against lower and upper bounds
+    return Bool
+end
+add_tfunc(isa, 2, 2, isa_tfunc, 0)
```

Comment:
Naming these tfuncs makes it so much nicer to experiment with them in the REPL. Is there a reason we don't format all of the tfuncs this way? Just conciseness?